readthedocs_ext/mixins.py

```diff
@@ -50,11 +50,17 @@ def copy_static_readthedocs_files(self):
                     context=ctx,
                 )
             else:
+                if sphinx.version_info < (1, 5):
```

Comment:
This check is already done in the outer if, never is going to reach this.

Reply:
Wow. How'd I miss that!